universal-login-relayer/lib/routes/config.ts

```diff
@@ -0,0 +1,30 @@
+import {Router, Request, Response} from 'express';
+import asyncMiddleware from '../middlewares/async_middleware';
+
+declare interface Config {
+  port: number;
+  jsonRpcUrl: string;
+  privateKey: string;
+  walletMasterAddress: string;
+  tokenContractAddress: string;
+  chainSpec: {
+    ensAddress: string;
+    chainId: number;
+  };
+  ensRegistrars: string[];
+}
```

Comment:
Extract to a separate file.

Reply:
Config:
```ts
require('dotenv').config();

const config = Object.freeze({
  legacyENS: true,
  jsonRpcUrl: process.env.JSON_RPC_URL,
  port: process.env.PORT,
  privateKey: process.env.PRIVATE_KEY,
  chainSpec: Object.freeze({
    ensAddress: process.env.ENS_ADDRESS,
    chainId: 0,
  }),
  ensRegistrars: [process.env.ENS_DOMAIN_1, process.env.ENS_DOMAIN_2, process.env.ENS_DOMAIN_3],
});

export default config;

export type Config = typeof config
export type ChainSpecConfig = Config['chainSpec']
```

Now:
export const network = (config : ChainSpecConfig) => async (req : Request, res : Response) => {
  res.status(200).send({ config });
};